packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -130,8 +131,10 @@ class GestureRecognizerFactoryWithHandlers<T extends GestureRecognizer> extends
 ///
 /// {@tool sample}
 ///
-/// This example turns the light bulb yellow when the "turn lights on" button is
-/// tapped by setting the `_lights` field:
+/// This example of a [Container] containing a black light bulb wrapped in a
+/// [GestureDetector].
```

Comment:
nit: this line break will not render. If you want a line break here you'd have to insert a blank line. If not, please just continue on the previous line (the latter is probably the best option).